src/cli/common.go

```diff
@@ -59,7 +59,10 @@ func (a *diffProcessCmd) run() error {
 	}()
 
 	// Load project remote and local state
-	projectState, ok := state.LoadState(projectManifest, logger, a.root.ctx, api, true)
+	stateOptions := state.NewOptions(projectManifest, api, a.root.ctx, logger)
```

Comment:
Metoda `state.LoadState` zacinala mat uz vela parametrov a bolo to neprehladne, preto som vsetky parametre obalil do struktury `state.Options`